src/components/Navigation.vue

```diff
@@ -43,7 +43,7 @@
 									!group.account.collapsed ||
 									!isCollapsed(group.account, item)
 							"
-							:key="item.databaseId"
+							:key="'mailbox-' + item.databaseId"
```

Comment:
looks like an unrelated change, no? and the databaseId should be unique enough. Did you just add this for clarity or to fix a bug?

Reply:
Ah, I forgot to mention that. Yea, this is unrelated indeed. I've been getting a duplicate key warning:

![Mail_-_Nextcloud](https://user-images.githubusercontent.com/459512/97970310-a0e22180-1dc1-11eb-9b0e-ce35e8bf7ab6.png)

Reply:
Ah now I see the issue. We display accounts and mailboxes. Hence there are indeed conflicts because they can have an overlap of IDs.

Good catch. Mind submtiting this as separate PR? :)

Reply:
Not at all :) I'll take care of the other comments first and will submit this separately afterwards.